source/mir/ndslice/topology.d

```diff
@@ -3924,43 +3923,298 @@ version(mir_test) unittest
 }
 
 /++
-Returns a slice that can be iterated by dimension. Transposes dimensions on top and then packs them.
+Returns a slice that can be iterated along dimension. Transposes other dimensions on top and then packs them.
 
-Combines $(LREF transposed) and $(LREF ipack).
+Combines $(LREF byDim) and $(LREF evertPack).
```

Comment:
Can you provide more detail here? What is the use case for alongDim versus byDim?